src/praisonai/praisonai/n8n/client.py

```
@@ -0,0 +1,221 @@
+        Returns:
+            True if connection successful
+        """
+        try:
+            session = self._get_session()
+            response = session.get(
+                f"{self.base_url}/healthz",
+                headers=self._get_headers(),
+                timeout=5.0
+            )
+            return response.status_code == 200
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)